src/orders/components/OrderDraftPage/OrderDraftAlert.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  ChannelUsabilityDataQuery,
+  OrderDetailsFragment,
+} from "@saleor/graphql";
+import { Alert, AlertProps } from "@saleor/macaw-ui";
+import React from "react";
+import { FormattedMessage, MessageDescriptor } from "react-intl";
+
+import { alertMessages } from "./messages";
+import { useAlertStyles } from "./styles";
+
+const getAlerts = (
+  order?: OrderDetailsFragment,
+  channelUsabilityData?: ChannelUsabilityDataQuery,
+) => {
+  const inactiveChannel = order && !order.channel.isActive;
```

Comment:
```suggestion
  const isChannelInactive = order && !order.channel.isActive;
```
or just `isChannelActive`